script/Common/00_00.txt

```diff
@@ -2025,7 +2025,7 @@ I'm discharged from the hospital soon after.
 [sha:66ea4c2a4f967b9eee47621d92ffc59af26cc42a]{
 -- Page 153, Offset 492.
 -- 退院したあと、僕は遠野の家ではなく他の家に預けられる事になった。
-When I leave, I'm not sent back to the Tohno mansion, but to another house.
+When I leave, I won't be sent back to the Tohno mansion, but to another house.
```

Comment:
Might revert this one, it makes it feel like he knows in advance what's coming when I don't think he does. Original is just present form but time has moved on.

Reply:
yeah, new version makes the chronology of this thought confusing